nextcloud/server
# Update improve devcontainer

* Resolves: #37169 

## Summary

Based on https://github.com/nextcloud/server/pull/37176.
Added many minor improvements to the DevContainer configuration so that one is now able to open the project in VSCode DevContainers / Codespaces and start working right away.

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits

## How to test

See `.devcontainer/README.md`. Tested on GitHub Codespaces, Docker for Linux and Docker Desktop for Windows. Just open, wait and you're done :smile:
